File: bench_arango/NET/min6api/Models/Retailer.cs
namespace min6api.Models;

public class Retailer {
  public int id {get; set;}
  public string GSTIN {get; set;} = string.Empty;
  public string Business_name {get; set;} = string.Empty;
  public string Contact_person {get; set;} = string.Empty;
  public int Contact_number {get; set;}
  public string Contact_address {get; set;} = string.Empty;
  public string Contact_emailId {get; set;} = string.Empty;
  public string Status {get; set;} = string.Empty;
  public int Outlet_limit {get; set;}
}

File: bench_pg/Go/shared/models/retailer.go
package model

type Retailer struct {
	//gorm.Model
	Id               int32  `json:"id"`
	GSTIN            string `json:"gstin"`
	Business_name    string `json:"business_name"`
	Contact_person   string `json:"contact_person"`
	Contact_number   int32  `json:"contact_number"`
	Constact_address string `json:"contact_address"`
	Contact_emailId  string `json:"contact_emailId"`
	Status           string `json:"status"`
	Outlet_limit     int32  `json:"outlet_limit"`
}
